### DISCUSSION TITLE
reload graphiql playground docs when schema changes during runtime

### DISCUSSION THREAD

**Discussion by ethdev279** [dotansimha/envelop#1465] [Q&A] [answered]
Is there anyway we can reload/refresh graphiql playground docs whenever there's a change in schema? 
I'm using envelop's `uselazyLoadedSchema` to select public and private schema's based on user throught context. 

```
plugins: [
    useLazyLoadedSchema(context => (context.isPrivateApiUser ? privateSchema : publicSchema))
  ]
```

Right now graphiql playground docs are not being updated when schema changes. Is there any way we can trigger refresh on schema change?

**Comment by n1ru4l** [comment, marked as answer]
duplicate of https://github.com/dotansimha/graphql-yoga/discussions/1535
